src/VisitorPattern/VisitorPattern.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
```

Comment:
When you can please update to netcoreapp2.0.

Reply:
done :smile: